src/scheduler/MpiWorld.cpp

```diff
@@ -1379,6 +1377,38 @@ double MpiWorld::getWTime()
     return t / 1000.0;
 }
 
+// This method is only used for testing
+std::vector<bool> MpiWorld::getInitedRemoteMpiEndpoints()
+{
+    if (!faabric::util::isTestMode()) {
+        SPDLOG_ERROR("Called method for testing outside test mode");
+        throw std::runtime_error("This method can only be called in testing");
+    }
```

Comment:
As mentioned above, I don't think there's a reason this shouldn't be called outside of a test if someone wants to.